gammapy/datasets/tests/test_simulate.py

```diff
@@ -847,7 +847,7 @@ def test_MC_ID_flag(model_alternative):
 
 
 @requires_data()
-def test_simulated_observation_maker(signal_model, tmp_path):
+def test_observation_event_sampler(signal_model, tmp_path):
```

Comment:
Should we make a test with a sky model that has a temporal model? Is it wise?

Reply:
it is tested for the MapDatasetEvent sampler that is used by the ObservationEventSampler, but I added a test for the ObservationsEventSampler.